### DISCUSSION TITLE
Looks like a good start

### DISCUSSION THREAD

**Discussion by jds11111** [tphakala/birdnet-go#29] [General]
I just installed and am running BirdNet-Pi. It is a great piece of kit, so am very happy to see someone taking over and modernizing it. What you have done so far is pretty impressive in the time frame. I saw the issues have ideas for work to come, but is there a vision for what the goals are in the project? For example, a key part for me is integration with automation, and the original software uses apprise to great effect here.  Or is the idea mainly to port over the functionality of BirdNet-Pi into a language that will be easier to maintain/develop?

**Comment by tphakala** [comment]
Thanks! I am planning adding notification system and also improving integration with external apps, infact code to run external scripts on detection already exists.
